ui/npm-install.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
-
```

Comment:
why?

Reply:
Accidental removal from making tons of changes to this file trying to get the build to work.